### PR TITLE
Adding Babel to API

### LINKED ISSUE #4259: [Maintenance] Add babel to api
### Description and related issues
We need to update the API package to use babel so that we can start using new modules and we can integrate with our Common package.

### This task is done when…

- [ ] move babel configs from web/e2e/common into the root directory so that they can all be shared
- [ ] add babel configs to override main configs for api

### PR DESCRIPTION
**Pull request by thetif**
Resolves #4259 

### Description
We needed to add babel to the api so that we can use the common package and so that we can upgrade other packages that use es6.

### Significant changes or possible side effects
This should only significantly change the build process. Once the app is built, as you are able to login and see APDs it should be fine

### This pull request is ready to review when

- [x] Pull request has been labeled, if applicable with feature, content, bug, tests, refactor

### This pull request can be merged when

- [x] Code has been reviewed by someone other than the original author
- [ ] QA has verified the functionality related to the change
- [ ] QA has verified the accessibility using tools such as screen readers and WAVE (accessibility evaluation tool)
- [ ] Product has approved the experience

### CODE REVIEW

File: api/unit-test.sh [hunk 1]
```diff
@@ -3,16 +3,18 @@
 export NODE_ENV=test
 
 docker-compose -f ../docker-compose.endpoint-tests.yml -p api up -d
+sleep 5
```
Comment (tbolt) [inline api/unit-test.sh:6]: why does this need to be added?

**Review by tbolt — APPROVED** [review]

File: api/unit-test.sh (hunk 1 above)
Comment (thetif) [reply api/unit-test.sh:6]: I don't know that it _has_ to be added, but sometimes it seems like the servers aren't fully up before it moves to the next step, so it's kind of insurance

**Comment by mirano-darren** [comment]
@thetif I am getting this error message when running the seed command.
image

**Comment by thetif** [comment]
@mirano-darren can you give it a try now, I think I resolved the issue.

**Review by mirano-darren — APPROVED** [review]

**Review by cthulhuplus — APPROVED** [review]
DevOps stuff looks good to me

**Pull request merged**